app/controllers/member_authentications_controller.rb

```diff
@@ -26,7 +26,8 @@ def create
 
     if auth.valid?
       flash[:notice] = I18n.t('member_registration.check_email')
-      path = follow_up_member_facing_page_path(page.slug, member_id: auth.member_id)
+      path = PageFollower.new_from_page(page, member_id: auth.member_id).follow_up_path
```

Comment:
I've noticed this pattern in a few places and I'm not sure I'm seeing the advantages over `PageFollower.new(page, options).follow_up_path`?

Looking at the `PageFollower` service [instance initialization](https://github.com/SumOfUs/Champaign/blob/development/app/services/page_follower.rb#L7-L17), I'm not sure if the initialization wrapper is unnecessary or if there's a benefit to it (I believe this applies to other places where the same pattern is used). 

```rb
def initialize(page, extra_params = nil)
  @page = page
  @plan = page.follow_up_plan
  ...
  @extra_params = extra_params.try(:symbolize_keys)
end
```